Source/FieldSolver/FiniteDifferenceSolver/MacroscopicEvolveM.cpp

```diff
@@ -119,8 +120,16 @@ void FiniteDifferenceSolver::MacroscopicEvolveM (
               // keep the interpolation. The IntVect is (1,0,0) to interpolate values to the x-face.
               Real mag_gamma_interp = MacroscopicProperties::macro_avg_to_face(i,j,k,amrex::IntVect(1,0,0),mag_gamma_arr);
               Real Gil_damp = PhysConst::mu0 * mag_gamma_interp
-                              * MacroscopicProperties::macro_avg_to_face(i,j,k,amrex::IntVect(1,0,0),mag_alpha_arr)
-                              / MacroscopicProperties::macro_avg_to_face(i,j,k,amrex::IntVect(1,0,0),mag_Ms_arr);
+                            * MacroscopicProperties::macro_avg_to_face(i,j,k,amrex::IntVect(1,0,0),mag_alpha_arr)
+                            / MacroscopicProperties::macro_avg_to_face(i,j,k,amrex::IntVect(1,0,0),mag_Ms_arr);
+              if (M_normalization == 0){
+                  Gil_damp *= MacroscopicProperties::macro_avg_to_face(i,j,k,amrex::IntVect(1,0,0),mag_Ms_arr)
+                            / std::sqrt( std::pow(M_xface(i, j, k, 0),2.0) + std::pow(M_xface(i, j, k, 1),2.0) + std::pow(M_xface(i, j, k, 2),2.0) );
+              }
+              else if (M_normalization < 0){
```

Comment:
@RevathiJambunathan should we move error checking for invalid parameter entries higher up? (like at the time we read them in to override a default value)  What is done in WarpX proper?